content/blog/2019-12-18-wasm-to-evm.md

```diff
@@ -0,0 +1,83 @@
++++ title = "Wasm-to-EVM binary translator" extra.author = "Hongbo Zhang" extra.author_link = "https://www.cs.cornell.edu/~hongbo/" extra.bio = """Hongbo Zhang is a first-year PhD student in computer science. He is interested in systems and computer architectures. He is also an okay archer shooting recurve bow.""" +++
```

Comment:
Hi! This will need to be reformatted to work in Zola. (You can try it out locally to see.)